lib/metadata/acl.js

```diff
@@ -12,11 +12,12 @@ const acl = {
         metadata.updateBucket(bucket.getName(), bucket, log, cb);
     },
 
-    addObjectACL(bucket, objectKey, objectMD, addACLParams, log, cb) {
+    addObjectACL(bucket, objectKey, objectMD, addACLParams, log, cb, params) {
```

Comment:
Yes, we have mentioned this in the previous step and I suggested to make it final step because changing the api involves searching for all references and modifying them in legacy code which is error prone.